core-domain/src/main/java/org/uniprot/core/uniprotkb/feature/UniProtKBFeature.java

```diff
@@ -0,0 +1,14 @@
+package org.uniprot.core.uniprotkb.feature;
+
+import org.uniprot.core.feature.Feature;
+
+public interface UniProtKBFeature extends Feature<UniprotKBFeatureDatabase, UniprotKBFeatureType> {
+
+    FeatureId getFeatureId();
+
+    AlternativeSequence getAlternativeSequence();
+
+    boolean hasFeatureId();
+
+    boolean hasAlternativeSequence();
```

Comment:
can't we provide default implementations ?

Reply:
The implementation requires an extra static map that is in the implementation class only.